packages/babel-helper-replace-supers/src/index.js

```diff
@@ -125,11 +125,18 @@ export default class ReplaceSupers {
     return t.callExpression(
       this.file.addHelper("set"),
       [
-        t.callExpression(
-          t.memberExpression(t.identifier("Object"), t.identifier("getPrototypeOf")),
-          [
-            this.isStatic ? this.getObjectRef() : t.memberExpression(this.getObjectRef(), t.identifier("prototype"))
-          ]
+        t.logicalExpression(
```

Comment:
I extracted the logic into a helper within babel-helper-replace-supers

I would have liked to share this method also with the transform-class package, but not sure where to put the helper function then.